examples/solo8_vanilla/observation_dump.py

```diff
@@ -19,6 +20,8 @@
 
   env.obs_factory.register_observation(obs.TorsoIMU(env.robot))
   env.reward_factory.register_reward(1,rewards.UprightReward(env.robot))
+  env.termination_factory.register_termination(
+    terms.TimeBasedTermination(100000000))
```

Comment:
We really need to make this magic number a variable somewhere

Reply:
For sure, made it into an issue: #37